prow/report/report.go

```diff
@@ -66,6 +66,22 @@ func prowjobStateToGithubStatus(pjState v1.ProwJobState) (string, error) {
 	return "", fmt.Errorf("Unknown prowjob state: %v", pjState)
 }
 
+const (
+	maxLen = 140
```

Comment:
consider add a comment/link to https://developer.github.com/v3/repos/deployments/#parameters-2

Reply:
Done